scripts/panopto.js

```diff
@@ -56,7 +59,7 @@ H5P.VideoPanopto = (function ($) {
           showtitle: false,
           autohide: true,
           offerviewer: false,
-          autoplay: !!options.autoplay,
+          autoplay: !!isAutoPlay,
```

Comment:
I believe this should be set to false, as all the others, since we handle the autoplay feature ourselves using an IntersectionObserver.